frameworks/youth-risk-assessment/questions/contact-number.yml

```diff
@@ -0,0 +1,7 @@
+type: text
+question: What is your contact number?
+description: Contact number
```

Comment:
I was thinking about this first time around, but should we use the [width](https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/tree/master/frameworks/person-escort-record#question-options) property here to set the width. Maybe `20` as some can contain maybe that `10`?